synctool/src/main/java/org/duracloud/sync/endpoint/DuraStoreSyncEndpoint.java

```diff
@@ -101,39 +102,17 @@ protected String getUsername() {
     }
 
     private void ensureSpaceExists() {
-        boolean spaceExists = false;
-        for (int i = 0; i < 10; i++) {
-            if (spaceExists()) {
-                spaceExists = true;
-                break;
-            }
-            sleep(300);
-        }
-        if (!spaceExists) {
+        if (!spaceExists()) {
             throw new RuntimeException("Could not connect to space with ID '" +
                                        spaceId + "'.");
         }
     }
 
-    private void sleep(long millis) {
-        try {
-            Thread.sleep(millis);
-        } catch (InterruptedException e) {
-            // Exit sleep on interruption
-        }
-    }
-
     private boolean spaceExists() {
         try {
             try {
-                Iterator<String> contents =
-                    contentStore.getSpaceContents(spaceId);
-                if (contents.hasNext()) {
-                    logger.warn("The specified space '" + spaceId +
-                                "' is not empty. If this space is being used for an " +
-                                "activity other than sync there is the possibility " +
-                                "of data loss.");
-                }
+                Map<String, AclType> acls =
```

Comment:
Since the return value isn't used for anything, there's really no need to capture it in a variable (and leaving it out means there's no need to import AclTye either).